Automattic/wp-calypso
# Site transfers: Add eligibility transfer check card

Related to https://github.com/Automattic/dotcom-forge/issues/2553

## Proposed Changes

This pr adds a check card for checking user eligibility for transferring a site.

## Testing Instructions

1. Apply D112291-code diff
2. Navigate to `Settings > General > Transfer your site`
3. Ensure that you see a component like the below:
![image](https://github.com/Automattic/wp-calypso/assets/497103/ee5ad834-3c4d-4e9f-8f37-10371fe198b4)
4. If the user is not found, you should see the following:
![image](https://github.com/Automattic/wp-calypso/assets/497103/581a6cce-1c7a-4d8a-982e-431003e0d4f8)
5. Ensure that after the user selection, the process finish successfully as before




## Pre-merge Checklist
